src/main/java/com/princexml/wrapper/Prince.java

```diff
@@ -111,6 +112,26 @@ private boolean convertInternal(List<String> inputPaths, String outputPath) thro
         return readMessagesFromStderr(process);
     }
 
+    /**
+     * Convert multiple XML or HTML files to a PDF file by reading an input list
+     * from a specified file.
+     * @param inputListPath The path to the input list file.
+     * @param outputPath The filename of the output PDF file.
+     * @return true if a PDF file was generated successfully.
+     * @throws IOException If an I/O error occurs.
+     */
+    public boolean convert(Path inputListPath, String outputPath) throws IOException {
+        List<String> cmdLine = getJobCommandLine("normal");
+        cmdLine.add(toCommand("input-list", inputListPath.toAbsolutePath().toString()));
```

Comment:
Should we be calling toAbsolutePath on any other paths we pass to Prince?

Reply:
Hmm. We don't use `Path` anywhere else in this wrapper - I did it mainly to disambiguate between overloaded methods, but perhaps the more sensible thing would be to maintain consistency with existing methods, i.e. just use a `String` and let Prince figure out the pathing.

I'll just name these methods `convertInputList` instead.

Reply:
Done. I've also clarified what an "input list" is.